### PR TITLE
chore: include new trg´s to default issue template

### PR DESCRIPTION
**Pull request by FaGru3n**
## Description

- add links to TRG 1.04 , 5.03, 7.08

## Pre-review checks

Please ensure to do as many of the following checks as possible, before asking for committer review:

- [X] DEPENDENCIES are up-to-date. [Dash license tool](https://github.com/eclipse/dash-licenses). Committers can open IP issues for restricted libs.
- [X] [Copyright and license header](https://eclipse-tractusx.github.io/docs/release/trg-7/trg-7-02) are present on all affected files

### CODE REVIEW

**Review by hzierer — APPROVED** [review]
LGTM

File: .github/ISSUE_TEMPLATE/system-team-qg-checklist.md [hunk 1]
```diff
@@ -30,6 +30,7 @@ Release Managemnet Reference Issue: <!-- Note: Add the related product RM issue
 - [ ] [TRG 1.01](https://eclipse-tractusx.github.io/docs/release/trg-1/trg-1-1) appropriate `README.md` 
 - [ ] [TRG 1.02](https://eclipse-tractusx.github.io/docs/release/trg-1/trg-1-2) appropriate install instructions either `INSTALL.md` or in `README.md`
 - [ ] [TRG 1.03](https://eclipse-tractusx.github.io/docs/release/trg-1/trg-1-3) appropriate `CHANGELOG.md`
+- [ ] [TRG 1.04]([https://eclipse-tractusx.github.io/docs/release/trg-1/trg-1-3](https://eclipse-tractusx.github.io/docs/release/trg-1/trg-1-4)) editable static files
```
Comment (tomaszbarwicki) [inline .github/ISSUE_TEMPLATE/system-team-qg-checklist.md]: Looks like the link is incorrect, should point to trg-1-4.

**Review by tomaszbarwicki — CHANGES_REQUESTED** [review]

File: .github/ISSUE_TEMPLATE/system-team-qg-checklist.md (hunk 1 above)
Comment (FaGru3n) [reply .github/ISSUE_TEMPLATE/system-team-qg-checklist.md]: Thanks for the hint, solved with [06f5afa](https://github.com/eclipse-tractusx/.github/pull/20/commits/06f5afa5034c222fa2dbff334f1b182c3bf3ed69)

**Review by tomaszbarwicki — APPROVED** [review]
LGTM

**Pull request merged**